Package.resolved

```diff
@@ -15,16 +15,16 @@
         "repositoryURL": "https://github.com/Carthage/Commandant.git",
         "state": {
           "branch": null,
-          "revision": "07cad52573bad19d95844035bf0b25acddf6b0f6",
-          "version": "0.15.0"
+          "revision": "2cd0210f897fe46c6ce42f52ccfa72b3bbb621a0",
+          "version": "0.16.0"
         }
       },
       {
         "package": "IBDecodable",
         "repositoryURL": "https://github.com/IBDecodable/IBDecodable.git",
         "state": {
           "branch": "master",
-          "revision": "eb8ebe5ec1ff87a07e4621510e23a5763525f014",
+          "revision": "3f1289222c5323605dede3e616e0f549bcb7c335",
```

Comment:
Current master

I think you are better to release with tags.

Reply:
I've just released 0.0.3 and using this.
https://github.com/IBDecodable/IBDecodable/releases/tag/0.0.3

Reply:
Thanks!